### ISSUE TITLE
タイトル生成に使用する基盤モデルを別途指定可能とする

### ISSUE THREAD

**Issue by okamoto-aws** [aws-samples/generative-ai-use-cases#287]
現状、タイトル生成に使用する基盤モデルは、`modelIds`の最初に記載されているものが使用される。
タイトル生成には軽量な`anthropic.claude-instant-v1`等が適するため、`modelIds`の記載にかかわらずタイトル生成用のモデルを選択できるようにする。
Note: instantは100Kトークンまで対応のため、モデルの入力トークン制限に合わせてプロンプトの量を調節する仕組みが必要？

**Comment by tbrand** [comment]
Instant V1 のアクティベートが必須になってしまうというのが課題で今の仕様になっています。やるとしたらデフォルトは今の動作で、特別に cdk.json などで指定できるようにする、という仕様でしょうか。

**Comment by okamoto-aws** [comment]
はい、cdk.jsonに、`titlePredictModelId`のようなオプションを用意しておき、デフォルトclaude v2にしておいても問題ないかと思います。
